scrapy_statsd_extension/__init__.py

```diff
@@ -33,14 +34,13 @@ def from_crawler(cls, crawler):
 
     def spider_opened(self, spider):
         if self.log_periodic:
-            self.log_task = task.LoopingCall(self.log_stats, spider)
+            self.log_task = utils.looping_call_task(self.log_stats, spider)
```

Comment:
I'm not sure it makes pulling this out in a function, since I can't imagine there's another method we could sub in here.

Reply:
Yeah, you're probably right here. Fixed it